README.md

```diff
@@ -34,6 +34,7 @@ To list the OPTIONS available use:
 * all bug fixes are welcome
 * testing with different .restore files and/or write tests
 * solve and remove the TODOs (if any exist)
+* improve hash performance without restarting long-running jobs
```

Comment:
This `Hacking` section / list was actually intended to mention some points how to improve this perl script (`analyze_hc_restore.pl`).
I think your added point is actually just an use case when to use this script.
In my opinion the added point doesn't really match the goal of this section ("how we could improve this perl script"), I think we should just skip this for this patch (because it doesn't really fit with the other code changes after all).... and maybe discuss in another issue if we really need to mention how `analyze_hc_restore` can be used or what it is for (if I understand your added point correctly)

Reply:
Ah thank you for pointing that out, I misinterpreted the section's intention and am glad you pointed that out. I agree with the removal and you did in fact understand correct my intention of mentioning a use case,

Reply:
Resolved in commit [7e12ab7](https://github.com/philsmd/analyze_hc_restore/pull/3/commits/7e12ab736ef0047cc309fce52a14f1337e23feb8)